apps/editing-toolkit/editing-toolkit-plugin/help-center/help-center-disconnected.js

```diff
@@ -11,7 +11,7 @@ function HelpCenterContent() {
 		<>
 			<Button
 				className="help-center"
-				onClick={ () => window.open( localizeUrl( 'https://wordpress.com/support/' ), '_blank' ) }
+				onClick={ () => window.open( localizeUrl( 'https://wordpress.com/help/' ), '_blank' ) }
```

Comment:
Can't we use `href`?

Reply:
Thanks! I didn't think the Button component could do that. I adjusted this.